modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+[id="network-observability-flowcollector-flowfilter-parameters_{context}"]
+= Flow filter configuration parameters
+The flow filter rules consist of mandatory and optional parameters.
+
+.Required configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`enable`
+| Set `enable` to `true` to enable eBPF flow filtering feature.
+
+|`cidr`
+| Provides the IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format. If you want to match against any IP, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
+
+|`action`
+| Describes the action to be taken for the flow filter rule. The possible values are `Accept` or `Reject`.
+
+* For the matching rule with the `Accept` action, this flow is allowed to be cached in the eBPF table, with the updated global metric `FlowFilterAcceptCounter`.
```

Comment:
```suggestion
* For `Accept`, the flow is cached in the eBPF table. The flow is updated with the global metric `FlowFilterAcceptCounter`.
```
I'm trying to simplify some of the notes here. I'm not sure if this changes the wording too much.